### ISSUE TITLE
Best cheats 

### ISSUE THREAD

**Issue by Zonemo001** [tomvita/MyNXCheats#11]
Thank you for all cheat you are the best

**Comment by Rogojinen** [comment]
Yes, thank you so much for the Zelda cheat and the update. I couldn't get any to work and I can't stand having to mini-micromanage resources to do anything. I can play in peace now.

**Comment by Zonemo001** [comment]
Thank you for the cheat 
 I hope for more cheat
